Updated --proxy-server= argument to Pyppeteer

dj2ball opened the pull request:
Now the string is  reformatted depending on whether the user:pass part of the string is present

davidteather commented:
Just looked at the PR right now you’re only passing in the user:pass into the puppeteer as a proxy. 

Shouldn’t it do proxy.replace(“user:pass@“, “”)?

dj2ball commented:
self.args.append("--proxy-server=" + proxy.split("://")[1].split(":")[1].split("@")[1] + ":" + proxy.split("://")[1].split(":")[2] 

returns 

'91.229.198.95:5432'

from this string passed into 'proxy':

'http://user:pass@91.229.198.95:5432'

Is this how you intended it?

davidteather commented:
Oh I didn’t read it correctly. It’s good except I think you need to add the https:// or http:// prior to the IP address. So if you add this before you should be good. proxy.split(“:”)[0] + “://“
